packages/@o3r/core/schematics/module/templates/base/package.json.template

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "<%= name %>",
+  "version": "0.0.0-placeholder",
+  "description": "<%= description || ('Otter module ' + name) %>",
+  "keywords": [
+    "apis-manager",
+    "otter"
+  ],
+  "scripts": {
+    "nx": "<%= isNxContext ? 'nx' : 'ng' %>",
+    "ng": "<%= runner %> nx",
+    "build": "<%= runner %> nx build apis-manager",
+    "prepare:build:builders": "yarn cpy 'collection.json' dist && yarn cpy 'schematics/**/*.json' dist/schematics",
+    "build:builders": "tsc -b tsconfig.builders.json --pretty"
+  },
+  "peerDependencies": {
+    "@angular/common": "<%= versions['angular'] %>",
+    "@angular/core": "<%= versions['angular'] %>",
+    "@o3r/dev-tools": "<%= otterVersion %>",
+    "@o3r/schematics": "<%= otterVersion %>"
+  },
+  "peerDependenciesMeta": {
+    "@angular-devkit/schematics": {
+      "optional": true
+    },
+    "@o3r/schematics": {
+      "optional": true
+    },
+    "@schematics/angular": {
+      "optional": true
+    }
+  },
+  "dependencies": {
+    "tslib": "^2.4.1"
+  },
+  "devDependencies": {
+    "@ama-sdk/core": "workspace:^",
```

Comment:
Why adding all this devDep by default ?

Reply:
I missed this one :).
The others are used to setup the build, lint and test actions of the module.
I was thinking that generating the basic commands for a module would more helpful than not doing it.